app/src/main/java/com/zulip/android/util/Constants.java

```diff
@@ -13,5 +13,6 @@
     //Default maximum time limit for editing message(Same as server)
     public final static int DEFAULT_MAXIMUM_CONTENT_EDIT_LIMIT = 600;
     public final static boolean DEFAULT_EDITING_ALLOWED = true;
+    public static String serverUrl = null;
```

Comment:
I guess I can't use _app.getServerURI()_ here because we need to store the url before it get flushed out. So an extra variable will be required.